### ISSUE TITLE
Require biometrics to access the app's data

### ISSUE THREAD

**Issue by Lppsoeht** [FossifyOrg/Voice-Recorder#63]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Voice-Recorder/issues) or [closed](https://github.com/FossifyOrg/Voice-Recorder/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Voice-Recorder/discussions) or [closed](https://github.com/FossifyOrg/Voice-Recorder/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Voice-Recorder/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Require biometrics to access app's data or access the app itself

### Why do you want this feature?

It would discourage a third party(physical person) from being able to access the app's data by knowing only the android's profile passcode.

**Comment by naveensingh** [comment]
Same as in Fossify Gallery, this would only prevent access to the app itself but not the recordings because they are saved in a publicly accessible storage. The third-party could use another app (e.g. an audio player) to access the recordings.

The only way this would work if we add an option to save recordings to the [app's private storage](https://github.com/FossifyOrg/Voice-Recorder/issues/62#issuecomment-2352803497) which is not accessible by third-party apps.

**Comment by naveensingh** [comment]
Yep, a new feature request about recording to private storage is welcome! The issue requesting encryption (#62) will likely to be closed in favor of this simpler approach.
